### PR TITLE
Allow specifying dns_alt_names for puppetdb

### LINKED ISSUE #233: Allow puppetdb.fqdns.alternateServerNames to be configured
## Use Case
Allow services outside of k8s to access puppetdb via HTTPS, otherwise it'll fail with SSL errors certificate not valid for x.y.z

## Describe the Solution You Would Like
Add puppetdb.fqdns.alternateServerNames parameter to add additional hostnames.

## Describe Alternatives You've Considered
No real alternatives except using http only which seems less than ideal.

## Additional Context
PR To follow

### PR DESCRIPTION
**Pull request by johannagnarsson**
This PR allows specifying additional dns_alt_names for puppetdb container.

This resolves #233.

### CODE REVIEW

File: templates/puppetdb-deployment.yaml [hunk 1]
```diff
@@ -111,7 +111,7 @@ spec:
             - name: PUPPETSERVER_PORT
               value: "{{ template "puppetserver.puppetserver-masters.port" . }}"
             - name: DNS_ALT_NAMES
-              value: {{ template "puppetdb.fullname" . }}
++              value: {{ template "puppetdb.san" . }}
```
Comment (Xtigyro) [inline templates/puppetdb-deployment.yaml]: I think we have here one space more than required.

**Review by Xtigyro — CHANGES_REQUESTED** [review]

File: templates/puppetdb-deployment.yaml (hunk 1 above)
Comment (johannagnarsson) [reply templates/puppetdb-deployment.yaml]: It was an actual plus sign, I've pushed a commit fixing it.

**Review by Xtigyro — APPROVED** [review]
@johannagnarsson Thank you!

**Comment by johannagnarsson** [comment]
@Xtigyro Is something needed to get this released ? Possibly just missing a `git tag` with the new version number?

**Comment by Xtigyro** [comment]
@johannagnarsson It's there now: https://github.com/puppetlabs/puppetserver-helm-chart/releases/tag/v9.5.2

Thanks for the reminder!

**Pull request merged**